src/data/scripts/hullmods/vayra_targeting_core.java

```diff
@@ -22,13 +21,13 @@ public class vayra_targeting_core extends BaseHullMod {
             HullMods.INTEGRATED_TARGETING_UNIT));
 
     // range bonus
-    public static float RANGE_BONUS = 1.5f; // multiplier for weapon range
+    public static float RANGE_BONUS = 50f; // multiplier for weapon range
 
     // handles applying stat bonuses
     @Override
     public void applyEffectsBeforeShipCreation(ShipAPI.HullSize hullSize, MutableShipStatsAPI stats, String id) {
-        stats.getBallisticWeaponRangeBonus().modifyMult(id, RANGE_BONUS);
-        stats.getEnergyWeaponRangeBonus().modifyMult(id, RANGE_BONUS);
+        stats.getBallisticWeaponRangeBonus().modifyPercent(id, RANGE_BONUS);
+        stats.getEnergyWeaponRangeBonus().modifyPercent(id, RANGE_BONUS);
```

Comment:
now why the fuck did he change multiplying by 1.5 to modifying by 50% percent. unnecessary change is unnecessary.

does the git blame say anything about why this was done?